include/openthread/platform/logging.h

```diff
@@ -110,7 +110,7 @@ extern "C" {
  * This type represents the log level.
  *
  */
-typedef uint8_t otLogLevel;
+typedef int otLogLevel;
```

Comment:
See the other comments. I think with that we can keep this to be `uint8_t`.

Reply:
Changing the type is not for reusing `aLogLevel`. When dynamic logging level is enabled, otDumpCertMeshCoP() is using `OT_LOG_LEVEL_NONE`(i.e. 0), https://github.com/openthread/openthread/blob/master/src/core/common/logging.hpp#L1903 will always be true, and some compiler will complain and fail.

Reply:
I see. Sounds good!